gen/dcos-config.yaml

```diff
@@ -379,6 +379,31 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
+  - path: /etc/dcos-metrics-config.yaml
+    content: |
+      #######################################
```

Comment:
nit: drop this comment... and probably the others. I don't think they add any value.